src/main/java/com/spring/api/transformer/UserDtoTransformer.java

```diff
@@ -0,0 +1,36 @@
+package com.spring.api.transformer;
+
+import com.spring.api.dto.UserDto;
+import com.spring.db.entity.user.User;
+
+public class UserDtoTransformer {
+    public static User transform(UserDto userDto) {
```

Comment:
method  `public static User transform(UserDto userDto, Long id) {`
and `public static User transform(UserDto userDto) {`
do the same thing, so try to rewrite it and optimize